### DISCUSSION TITLE
练习

### DISCUSSION THREAD

**Discussion by huangqiu1** [ruanqizhen/labview_book#74] [Q&A]
大神，可以把每章的结尾练习例程开拓出来吗，初学者，有些地方想借鉴一下

**Comment by ruanqizhen** [comment]
如果是参考借鉴，可以使用书中的示例，或者 LabVIEW 自带的示例。这些示例 VI 还是非常多的，基础知识都已经被覆盖到了。另外，我将保留一些未提供答案的问题，是为了鼓励大家亲自动手编程，而不仅仅只是阅读别人的代码。
